edx_proctoring/views.py

```diff
@@ -946,3 +949,36 @@ def post(self, request):
                          request.data,
                          backend=provider)
         return Response('OK')
+
+
+class InstructorDashboard(AuthenticatedAPIView):
+    """
+    Redirects to the instructor dashboard for reviewing exams on the configured backend
+    """
+    @method_decorator(require_course_or_global_staff)
+    def get(self, request, course_id, exam_id=None):
+        """
+        Redirect to dashboard for a given course and optional exam_id
+        """
+        if exam_id:
+            exam = get_exam_by_id(exam_id)
+        else:
+            found_backend = None
+            for exam in get_all_exams_for_course(course_id, True):
```

Comment:
Do we need to deal with a no-exam course?

Reply:
I added a test for that condition, and will return 404 for courses with no exams.